conda-recipe/meta.yaml

```diff
@@ -22,6 +22,7 @@ requirements:
       - numpy
       - ophyd
       - pandas
+      - pyqt <5.15.0
```

Comment:
Should this be in `run-constrained`? pyqt isn't actually a dependency, I think?
Not a show stopper regardless

Reply:
A good point.  A quick search confirms this.